libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -209,6 +209,15 @@ const AP_Param::GroupInfo AP_Airspeed::var_info[] = {
     // @Units: m/s
     // @User: Advanced
     AP_GROUPINFO("_WIND_WARN", 23, AP_Airspeed, _wind_warn, 0),
+
+    // @Param: _WIND_GATE
+    // @DisplayName: Re-enable Consistency Check Gate Size
+    // @Description: Number of standard deviations applied to the re-enable EKF consistency check that is used when ARSPD_OPTIONS bit position 3 is set. Larger values will make the re-enabling of the airspeed sensor faster, but increase the likelihood of re-enabling a degraded sensor. The value can be tuned by using the ARSP.TR log message by setting ARSP_WIND_GATE to a value that is higher than the value for ARSP.TR observed with a healthy airspeed sensor. Occasional transients in ARSP.TR above the value set by ARSP_WIND_GATE can be tolerated provided they are less than 5 seconds in duration and less than 10% duty cycle.
+    // @Description{Copter, Blimp, Rover, Sub}: This parameter and function is not used by this vehicle.
+    // @Range: 0.0 10.0
+    // @User: Advanced
+    AP_GROUPINFO("_WIND_GATE", 26, AP_Airspeed, _wind_gate, 5.0f),
```

Comment:
I think this skips param 25, looks like 24 is `"_DEVID"`

Reply:
param 25 is 2_DEVID. I'll update the warning on line 304